learning/features/inputs/inputevent.rst

```diff
@@ -41,7 +41,7 @@ received input, in order:
    "input_event" will be emitted (this function is re-implementable by
    script by inheriting from it). If the control wants to "consume" the
    event, it will call :ref:`Control.accept_event() <class_Control_accept_event>` and the event will
-   not spread any more.
+   not spread any more. Events that are not consumed will "bubble **up**", to :ref:`Control <class_Control>`'s ancestors. :ref:`Control.mouse_filter <class_Control_mouse_filter>` property can be used to control this behavior for mouse events.
```

Comment:
Avoid passive voice, and I'd say propagate instead of bubble.
Suggestion: 
`Events that are not consumed will propagate **up** to :ref:`Control <class_Control>`'s ancestors.  Use the :ref:`Control.mouse_filter <class_Control_mouse_filter>` property to change this behavior for mouse events.`